### ISSUE TITLE
Accessibility Issue: Ensure buttons have discernible text

### ISSUE THREAD

**Issue by bettirosengugi** [microsoftgraph/microsoft-graph-explorer-v2#333]
**Describe the bug**
**Buttons:**

- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"
- Element has no title attribute or the title attribute is empty

**To Reproduce**

Use the Accessibility Insights tool to Fast Pass test to make the test and see the accessibility issues. 

Pay attention to the Preview Button, but ensure all buttons are checked.
**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/4668452/72338871-727e4300-36d6-11ea-9a58-59f989e2c3ce.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
